tests/README.md

```diff
@@ -0,0 +1,19 @@
+# e2e testing 
```

Comment:
No need for this file, it's already defined in a GitHub issue.

Instead please add a new `e2e` entry in the `scripts` section of `package.json` that runs the suite.